examples/cli/CMakeLists.txt

```diff
@@ -10,10 +10,19 @@ add_executable(${TARGET}
     vad.h
 )
 
-find_package(SDL2)
-if (SDL2_FOUND)
-    target_link_libraries(${TARGET} PRIVATE SDL2::SDL2)
-    set_source_files_properties(playback.cpp PROPERTIES COMPILE_FLAGS -DSDL2_INSTALL=1)
+find_package(SDL2 QUIET)
```

Comment:
Some users want this to remain an optional dependency that isn't linked into the final build. See whisper.cpp's `option(WHISPER_SDL2 "whisper: support for libSDL2" OFF)`, and please adapt that to this file

Reply:
I added `TTS_CLI_SDL` that you can turn off for this use case